fdbclient/ServerKnobs.cpp

```diff
@@ -164,6 +164,15 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PRIORITY_SPLIT_SHARD,                                  950 ); if( randomize && BUGGIFY ) PRIORITY_SPLIT_SHARD = 350;
 
 	// Data distribution
+	init( AVAILABLE_SPACE_PIVOT_RATIO,                         0.6 );
+	init( CPU_PIVOT_RATIO,                                     0.9 );
+	// In order to make sure GetTeam has enough eligible destination team:
+	ASSERT_GT(AVAILABLE_SPACE_PIVOT_RATIO + CPU_PIVOT_RATIO, 1.0 );
```

Comment:
The teams which have value **lower than** the pivot team are eligible. In your case, there are at least 80% of teams are eligible.

Reply:
Ah, I've figured it out. The key point here is "Eligible teams space > space ratio && Eligible teams CPU < CPU ratio" since space is count for AVAILABLE and CPU is for usage...